membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -60,17 +60,16 @@ class AttributeController(attributesFromZuora: AttributesFromZuora, commonAction
   }
 
   private def lookup(endpointDescription: String, onSuccessMember: Attributes => Result, onSuccessSupporter: Attributes => Result, onNotFound: Result, sendAttributesIfNotFound: Boolean = false) = {
-    AuthAndBackendViaIdapiAction(ContinueRegardlessOfSignInRecency).async { implicit request: AuthAndBackendRequest[AnyContent] =>
-      if(endpointDescription == "membership" || endpointDescription == "features") {
```

Comment:
has the need for this logging gone away now?

Reply:
Oh good spot; this was a merge accident! I will put it back